doc/api/console.markdown

```diff
@@ -72,15 +72,15 @@ object. This is useful for inspecting large complicated objects. Defaults to
 - `colors` - if `true`, then the output will be styled with ANSI color codes.
 Defaults to `false`. Colors are customizable, see below.
 
-### console.time(timerName)
+### console.time(label)
```

Comment:
We recently changed this in https://github.com/nodejs/node/pull/3166 to match [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Console/time). I'd say we better ought to try changing the spec (which is linked on MDN, ironically).